src/main/java/games/strategy/net/MacFinder.java

```diff
@@ -175,19 +175,19 @@ private static String executeCommandAndGetResults(final String command) {
     }
     try {
       final StringBuilder builder = new StringBuilder();
-      final BufferedReader in = new BufferedReader(new InputStreamReader(p.getInputStream()));
-      while (true) {
-        try {
-          final String line = in.readLine();
-          if (line == null) {
+      try (final BufferedReader in = new BufferedReader(new InputStreamReader(p.getInputStream()))) {
+        while (true) {
```

Comment:
Not really related to the other code, but what about using a Scanner instead? This way the code could be shortened to
```
final StringBuilder builder = new StringBuilder();
try (final Scanner scanner = new Scanner(p.getInputStream())) {
	while (scanner.hasNextLine()) {
		builder.append(scanner.nextLine()).append("\r\n");
	}
	return builder.toString()
}
```
Or you could simply use `return IOUtils.toString(p.getInputStream())`, which should also work